app/Http/routes.php

```diff
@@ -11,14 +11,38 @@
 |
 */
 
+<<<<<<< Updated upstream
 /*-------------------------
+=======
+
+/*
+>>>>>>> Stashed changes
  * Routes Authentication
- * ------------------------
  */
 
-//Routes to pages with authentication
-Route::auth();
+Route::group(['middleware' => ['web']], function() {
```

Comment:
Why are you adding this back it will cause the web middleware to be run twice messing with error messages?